releasenotes/notes/0.5/experiment_data_fixes-f69c3569a8ba1342.yaml

```
@@ -0,0 +1,17 @@
+---
+upgrade:
+  - |
+    1. :meth:`.ExperimentData.save` now uses the multithreading capability
+    of the experiment service to enable faster saving times.
+    2. :class:`.ExperimentData` now supports the new method 
+    :meth:`.ExperimentData.get_service_from_provider` enabling the automatic
+    setting of the experiment database service via passing the provider.
+    3. The `end_datetime` property of :class:`.ExperimentData` is now being
+    set to the latest time a successful job terminated.
+    4. :meth:`.ExperimentData.save` can now raise exceptions when saving
+    fails, unless the `suppress_errors` flag is set (on by default)
+
+fixes:
+  - |
+    1. Fixed a bug in :meth:`.ExperimentData._add_job_data` where the wrong
```

```suggestion
    Fixed a bug in :meth:`.ExperimentData._add_job_data` where the wrong
```
This isn't helpful for users because the internal method isn't exposed in the documentation. It's better to describe what the user would have been experiencing and what the updated behavior is.